src/components/ContactForm.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+
+class ContactForm extends React.Component {
+
+  constructor() {
+    super();
+
+    this.state = {
+      name: '',
+      picture: '',
+      occupation: ''
+    };
+  }
+
+  handleNameChange(event) {
+    this.setState({
+      name: event.target.value
+    });
+  }
+
+  handleOccupationChange(event) {
+    this.setState({
+      occupation: event.target.value
+    });
+  }
+
+  handlePictureChange(event) {
+    this.setState({
+      picture: event.target.value
+    });
+  }
```

Comment:
Did you see the extension in the curriculum for how to write the handler logic in one function?